src/main/java/dev/espi/protectionstones/commands/ArgAdminHelp.java

```diff
@@ -16,40 +16,149 @@
 package dev.espi.protectionstones.commands;
 
 import dev.espi.protectionstones.ProtectionStones;
-import net.md_5.bungee.api.chat.ComponentBuilder;
+import net.md_5.bungee.api.chat.BaseComponent;
+import net.md_5.bungee.api.chat.ClickEvent;
 import net.md_5.bungee.api.chat.HoverEvent;
 import net.md_5.bungee.api.chat.TextComponent;
 import org.bukkit.ChatColor;
 import org.bukkit.command.CommandSender;
 
 public class ArgAdminHelp {
 
-    private static void send(CommandSender p, String text, String info) {
-        TextComponent tc = new TextComponent(text);
-        tc.setHoverEvent(new HoverEvent(HoverEvent.Action.SHOW_TEXT, new ComponentBuilder(info).create()));
-        p.spigot().sendMessage(tc);
+    private static void send(CommandSender p, String text, String info, String clickCommand, boolean run) {
+        // Create the main text component from legacy text.
+        BaseComponent[] mainComponents = TextComponent.fromLegacyText(text);
+        TextComponent mainText = new TextComponent("");
+        for (BaseComponent component : mainComponents) {
+            mainText.addExtra(component);
+        }
+
+        // Create the hover event from the info text, add click event after
+        BaseComponent[] hoverComponents = TextComponent.fromLegacyText(info);
+        mainText.setHoverEvent(new HoverEvent(HoverEvent.Action.SHOW_TEXT, hoverComponents));
+        //toggle for running on mouse click, currently disabled
```

Comment:
```suggestion
        // Toggle for running on mouse click
```

It is implemented here, so it isn't disabled